app/templates/login.hbs

```diff
@@ -0,0 +1,141 @@
+<AuApp>
+  <AuMainContainer as |m|>
+    <m.content @scroll={{true}}>
+      <AuContentHeader
+        @titlePartOne="Vlaanderen"
+        @titlePartTwo="is lokaal bestuur"
+      >
+        <img
+          sizes="50vw"
+          src={{root-url
+            "/assets/images/loket-header-1600-85a5251e74ef6ac8491b2f228d2c90da.jpg"
+          }}
+          srcset={{@srcSet}}
```

Comment:
It's not showing the image for me.

Reply:
It needs to be copied into the public assets folder first.

Reply:
and the `@srcSet` will be undefined here. It needs to be hardcoded path similar to the source.